theme/sass/components/_card-component.scss

```diff
@@ -95,6 +103,29 @@
                         font-size: $font-medium;
                     }
                 }
+
+                &--horizontal {
+                    .card-image-wrapper {
+                        height: 350px;
```

Comment:
Setting the height like this isnt exactly best practice, consider revising this using this. https://www.uwl.ac.uk/themes/custom/rutherford/pattern-lab/public/?p=molecules-card-basic

Reply:
done.